### ISSUE TITLE
Colemak layout default Back/forward bindings do not work (fall through to firefox actions)

### ISSUE THREAD

**Issue by 0ion9** [mortie/mouseless-plugin#20]
The default setting for 'Go Back / Forward' is '<Control>N/<Control>O'. However, after saving settings and restarting Firefox, these keybinds still do not take effect. Pressing ^N creates a new window, and pressing ^O opens an 'open file' dialog; these are the normal Firefox actions for these shortcuts.

Binding to <Control><Shift>N|O has similar problems ("Open settings page in new tab" / "Open bookmarks library")

For now I have shifted the 'move tab' shortcuts onto 'go back / forward', and have not set the 'move tab' shortcuts to anything (the first ideas I had for non-conflicting binds, <Alt>N|O and <Control><Alt>N|O , simply don't work). That's ok, I don't really move tabs much.

In summary: *Default keybinds for Colemak layout 'Go Back/ Forward' actions do not work*, and I sympathize if you want to say 'that's really Firefox's fault', but it is my opinion that these binds should be omitted from the config for now.

mouseless version: 0.13.0 from the addons.mozilla.org site, installed today.
Firefox version: 120.0
OS: Arch Linux x86_64, last updated 2023-11-22 (today as of the time I'm writing this bug report)
